### ISSUE TITLE
PCB and BOM for 1.1.5 Silent Board?

### ISSUE THREAD

**Issue by fivesixzero** [Creality3DPrinting/Ender-3#45]
While working with my new V1.1.5 Silent Board I’ve found some subtle but important dIfferences between the 1.1.2 board detailed in this repo. It’d help a lot, especially with any future repairs, to have an accurate BOM and layout for this board.

The main difference I’ve noticed is the use of R150s for the sense resistor pairs instead of R100s. Based on the TMC2208 data sheet it looks like this effectively caps the output of the motors at about 15% below the max RMS for the 2208 part at 1.28 Amps (against a 1.4 A RMS max in the 220x data sheet).

That seems like a wise decision with the new drivers, helping to avoid burning out the chips. With this design, if Vref is set to 2.5V or higher it would avoid an over-current situation in the driver. Which is good.

Given that minor change I’m also curious what else has changed, if anything, between the 1.1.2, 1.1.4 and 1.1.5 boards as well as what’s different between the 1.1.5 and the 1.1.5 Silent Board beyond the drivers and the sense resistors.

I’m also curious about the stability of the Vref coming out of the trim pots. The 2208 data sheet has some strong warnings about layout and design for stable analog Vref as well as sense resistor grounding. Reviewing the design without reverse engineering the physical board to verify things would require updated design and BOM data.

TL;DR

1. What changed in BOM or layout between the 1.1.2, 1.1.4, and 1.1.5 board revisions?
2. What changed in BOM or layout between the “normal” and “silent” board revisions?

**Comment by rickyrockrat** [comment]
They removed the FTDI chip and replaced it with a CH340G. Unfortunately, the USB port STILL powers that LCD backlight when 24V is off. It would be very nice to see a PCB layout so I can fix that like I did on the 1.1.3 version.

Turns out, It is even easier on the 1.1.5. Pin 16 of U1 (CH340G) goes to a via, then the decoupling cap C46. That via goes to the bottom side and is a stub off of the +5V trace. Cut the trace on the bottom side between the via and the main +5V trace, then solder a wire from the Via on the +5V USB trace (right next to the pin) to the side of the cap connected to pin 16. Then remove DP2, a diode that provides power to the rest of the board.
Problem solved. I don't know why Creality keeps on doing this. It really is annoying.

**Comment by Xinayder** [comment]
They should publish it as RAMPS is open-source and it requires source disclosure thanks to its license, they haven't published the source code for their silent board yet.
